include/tst_safe_pthread.h

```diff
@@ -18,6 +18,8 @@
 #ifndef TST_SAFE_PTHREAD_H__
 #define TST_SAFE_PTHREAD_H__
 
+#include <pthread.h>
+
 int safe_pthread_create(const char *file, const int lineno,
 			pthread_t *thread_id, const pthread_attr_t *attr,
 			void *(*thread_fn)(void *), void *arg);
```

Comment:
I wonder if this is a good idea, that way all testcases would include pthread.h. Maybe we should just remove it from the tst_test.h and include it in the few testcases that use it separately. And we should do that in a separate patch as well.